packages/rocketchat-i18n/i18n/assistifyAI.de.i18n.yml

```diff
@@ -10,6 +10,8 @@ Assistify_AI_Smarti_Domain: Smarti Client-Name
 Assistify_AI_Smarti_Domain_Description: Trage hier den Smarti Client-Name ein, der zu dieser Rocket.Chat Instanz gehört.
 Assistify_AI_Smarti_Auth_Token: Smarti Client-Token
 Assistify_AI_Smarti_Auth_Token_Description: Trage hier den Smarti Client-Token ein, damit sich Rocket.Chat gegenüber Smarti authentifizieren kann.
+Assistify_AI_Smarti_Widget_i18n: Texte des Smarti-Widgets
+Assistify_AI_Smarti_Widget_i18n_Description: "Ein JSON-Objekt, {KEY: {LANGUAGE1: TEXT1, LANGUAGE2: TEXT2}}"
```

Comment:
There should be a list of possible `KEY's displayed, otherwise, an admin can't know what to put there.

Available keys:
```
"widget.conversation.answer.title"
"widget.conversation.answer.title_msg"
"widget.latch.answer.title"
```
Example:
```
{
  "widget.conversation.answer.title": {
    "de":"Ich habe eine passende Konversation gefunden:",
    "en":"I found a similar conversation:"
  }
}
```